alf/environments/gym_wrappers.py

```diff
@@ -573,7 +585,10 @@ def _space_bounds(space):
             else:
                 return min_v, max_v
 
-        self.bounds = alf.nest.map_structure(_space_bounds, self.action_space)
```

Comment:
Should have commented in docstring that `self.action_space` is not really its valid action space as we can always accept any action and clip it to the desired range. For now we just keep it the same with the action space of the underlying `self.env`.

Reply:
commented

Reply:
If self.action_space still uses Gym's Dict and Tuple, will the Network class correctly handle it?